code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -98,7 +98,6 @@
 //returns 1 if the holder should continue reactiong, 0 otherwise.
 /datum/reagents/proc/process_reactions()
 	if(!my_atom || !my_atom.loc || my_atom.flags & NOREACT)
-		equalize_temperature()

```

Comment:
Why did you remove this?

Reply:
I thought it was originally part of the problem, but it wasn't, however upon review it should not be there because of ```my_atom.flags & NOREACT```